### ISSUE TITLE
Focus login input when opening the login and register page

### ISSUE THREAD

**Issue by andidev** [jhipster/generator-jhipster#954]
The login input is not focused when opening the login page. This is a bit anoying when the user need to click it with the mouse before typing the login name and password.

I could fix this in a pull request if wanted. Seems that there is a need for adding the an autofocus directive to acomplish this which can be found in bower https://github.com/ianwalter/ng-autofocus.

**Comment by andidev** [comment]
Actually the Home button is supposed to be active when the home page is open and this is the case when visiting https://senlima-dev.herokuapp.com/#/.
If the users clicks Account Button ---> Authenticate then Account Button will be active instead.

**Comment by MartinLichtblau** [comment]
Ahh, ok. But I find it kinda unintuitive like that. Because then two buttons can be higlighted at the same time. Normally you give feedback only on the element which is currently selected. And if I click on a dropdown button I stay on the "Home" page, so it's highlighted, and at the same time the dropdown button which I have selected is highlighted. 
![jhipste_navbar2](https://cloud.githubusercontent.com/assets/4956334/5682918/f10996fe-9823-11e4-98e3-0608bf0cb55c.PNG)

**Comment by andidev** [comment]
Agree with you, howerever I this is rather a bootstrap issue since it works this way in bootstrap. Take a look at the navbar in the [bootstrap docs](http://getbootstrap.com/javascript/#scrollspy-examples). It seems to have the same issue.
An alternative is not to highlight the button that is accosiated with currently open page. Or maybe hange the active color so that the colors differ (if the bootstrap css classes allow that).Though this is a completely different issue then #954, so you should create a new issue for that.

**Comment by jdubois** [comment]
Yes, it's how Bootstrap works: I'm not saying it's good, just that we use the default settings, which is our philosophy

**Comment by andidev** [comment]
Though this issue was not about that sidetrack that @MartinLichtblau reported. It's about focusing the login input when opening the authentication page so that the user does not need to click it manually with the mouse before starting to type its login credentials. Is it possible to reopen? I could provide a PR for this.

**Comment by jdubois** [comment]
Oh, sorry, yes I'm reopening it.
